components/AudioRecorder/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Image, StyleSheet, Text, TouchableHighlight, View } from 'react-native';
+import _ from 'lodash-fp';
+
+const micIcon = require('./../../assets/ic_mic_white_24dp_2x.png');
```

Comment:
wait for @FalahShazib 's branch to merge but we're going to be handling icons in a diff way :) 

she's making an Icon component that you can use in place of this

Reply:
Okay sounds great!